src/modules/battery_status/analog_battery.cpp

```diff
@@ -127,6 +127,4 @@ AnalogBattery::updateParams()
 			param_set_no_notification(_analog_param_handles.a_per_v_old, &_analog_params.a_per_v_old);
 		}
 	}
-
-	Battery::updateParams();
```

Comment:
Why is this removed?

Reply:
@bkueng can you check if my commit message here makes sense?
https://github.com/PX4/Firmware/pull/14490/commits/d819c52ff12e599a259663f05e46269973ef1c41

Reply:
But this is not how inheritance works, or rather the pattern that is used here. What we have here is a subclass overriding a parent's implementation, because it has some customizations, and then calls the parent's method.

Reply:
Ok, in that case I'll revert this.